### ISSUE TITLE
`import` module without extension causes compiling error

### ISSUE THREAD

**Issue by yaquawa** [graphql/graphql-js#2826]
I'm getting errors like this when compiling

```
error  in ./node_modules/graphql/language/printer.mjs

Module not found: Error: Can't resolve './visitor' in '/Users/Documents/htdocs/node_modules/graphql/language'
Did you mean 'visitor.mjs'?
```

in the `graphql/language/index.mjs`,

if I change this

```
export { visit, visitInParallel, visitWithTypeInfo, getVisitFn, BREAK } from './visitor';
```

to this
```
export { visit, visitInParallel, visitWithTypeInfo, getVisitFn, BREAK } from './visitor.mjs';
```
 will work.

**Comment by IvanGoncharov** [comment]
@yaquawa What package version do you use?
Also, do you get this package through NPM?

**Comment by yaquawa** [comment]
@IvanGoncharov 

> What package version do you use?

I'm using version 14.0.0

> do you get this package through NPM?

Yes, as a dependency of [aws-amplify](https://github.com/aws-amplify/amplify-js).

**Comment by IvanGoncharov** [comment]
@yaquawa This issue was fixed in `15.0.0` as #2379
